sql/core/src/main/scala/org/apache/spark/sql/execution/streaming/IncrementalExecution.scala

```diff
@@ -271,8 +343,12 @@ class IncrementalExecution(
    * updated metadata.
```

Comment:
Should we update this comment now ?

Reply:
The existing explanation is still valid. If we want to be verbose then I'll explain the new change into comment.